services/secondary-storage/server/src/main/java/org/apache/cloudstack/storage/template/DownloadManagerImpl.java

```diff
@@ -596,54 +594,54 @@ public String downloadPublicTemplate(long id, String url, String name, ImageForm
             File file =
                     ResourceType.TEMPLATE == resourceType ? _storage.getFile(tmpDir + File.separator + TemplateLocation.Filename) : _storage.getFile(tmpDir + File.separator +
                             "volume.properties");
-                    if (file.exists()) {
-                        if(! file.delete()) {
-                            LOGGER.warn("Deletion of file '" + file.getAbsolutePath() + "' failed.");
-                        }
-                    }
+            if (file.exists()) {
+                if(! file.delete()) {
+                    s_logger.warn("Deletion of file '" + file.getAbsolutePath() + "' failed.");
```

Comment:
moving away from the naming convention, (also i think it is not declared like this)